### PR TITLE
Functional test: Item checkin

### PR DESCRIPTION
**Pull request by thofmann**
### Summary

This creates a functional test for navigating to a student and checking in an item.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Review the test for comprehensiveness.
2. Run and watch the test.

### Links

- [TFF-127](https://msoese.atlassian.net/browse/TFF-127)

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -179,5 +179,6 @@ module.exports = function(grunt) {
     grunt.registerTask('build', ['clean:dist', 'babel:dist', 'browserify:dist', 'stylus', 'copy', 'inline']);
     grunt.registerTask('lint', ['eslint']);
     grunt.registerTask('test', ['lint', 'build', 'clean:test', 'babel:test', 'mochacli']);
+    grunt.registerTask('lintless-test', ['build', 'clean:test', 'babel:test', 'mochacli']);
```
Comment (Cafferoq) [inline Gruntfile.js]: Woot!

File: test/functional/item-checkout.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                return mockServer.stop();
+            });
+        }
+    });
+
+    beforeEach(() => {
+        mockServer.clearExpectations();
+    });
+
+    it('loads the homepage', () => {
```
Comment (Cafferoq) [inline test/functional/item-checkout.js]: Not sure if this test should be a part of the check-in functional tests. but it's not that big of a deal.

Comment (thofmann) [inline test/functional/item-checkout.js]: Good point. I'm not sure either.

Comment (mattmahn) [inline test/functional/item-checkout.js]: I don't think it should be; that deleted file would be a much better place.

File: src/index.js [hunk 3]
```diff
@@ -19,8 +19,16 @@ function createWindow() {
         window.webContents.openDevTools();
     }
 
+    // Capture an optional port from the command line args
+    let port = 80;
+    process.argv.forEach(arg => {
+        if (arg.substring(0, 7) === '--port=') {
```
Comment (thofmann) [inline src/index.js]: I considered doing this. Will change it now.

File: test/functional/item-checkout.js (hunk 2 above)
Comment (thofmann) [inline test/functional/item-checkout.js]: :+1: I'll remove it now.

**Pull request merged**